projects/bp-gallery/environments/.env.production

```diff
@@ -0,0 +1,3 @@
+REACT_APP_API_BASE=https://bp.bad-harzburg-stiftung.de/api
+REACT_APP_CONTACT_MAIL=archiv@bad-harzburg-stiftung.de
+REACT_APP_SENTRY_DSN=https://315a3c261a1c4b5d81de2241b7e7cfc9@o1243479.ingest.sentry.io/6458997
```

Comment:
Is this adress likely to change?

Reply:
Also: could you add `REACT_APP_ADVANCED_SEARCH=false` here when #115 is merged? Thanks! :)

Reply:
> Is this adress likely to change?

No, but it should definitely be different for different environments (dev, staging, prod) to be able to discern where errors come from.